tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -435,4 +435,105 @@ virtual void TearDown() {
         EXPECT_EQ(0, verAdvances[i].height); // Yes actually!
     }
 #endif
+}
+
+TEST(CTFont, GetBoundingBox) {
+// Font names differ slightly between platforms
+// Windows may have slightly different versions of the same fonts compared to the reference platform
+#if TARGET_OS_WIN32
+    CFStringRef name = CFSTR("Arial Italic");
```

Comment:
too much of a stretch at this point, maybe when the need is more apparent.

---
In reply to: [83340634](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r83340634) [](ancestors = 83340634)